xgi/drawing/draw.py

```diff
@@ -739,7 +740,30 @@ def _color_arg_to_dict(arg, ids, cmap):
 
     Parameters
     ----------
-    arg : str, dict, iterable, or NodeStat/EdgeStat
+    arg : There are several different valid arguments. They are:
+
+        Single color values
+
+        * str
+        * 3- or 4-tuple
+
+        Iterable of colors
+
+        * numpy array of color values as described above
+        * list of color values as described above
+        * dict of color values as described above
```

Comment:
for brevity (I would even maybe put numpy array and list on the same line): 
```suggestion
        Iterable of colors (each specified as above)

        * numpy array
        * list
        * dict
```
one question: in the `dict` case, I guess values are used? Keys should refer to node/edge IDs?

Reply:
Yep! I added a little example of the format.